lib/utils.js

```diff
@@ -24,7 +24,9 @@ module.exports = {
   makeStringPayload
 };
 
-let contents = fs.readFileSync(config.get('template'), 'utf-8');
 function makeStringPayload() {
-  return faker.fake(contents);
+  if (!this.cache) {
```

Comment:
use `makeStringPayload.cache` to prevent problems when setting  `this` context outside this module